javaagent-exporters/logging/src/main/java/io/opentelemetry/javaagent/exporters/logging/LoggingExporter.java

```diff
@@ -25,13 +25,14 @@ public LoggingExporter(String prefix) {
   public CompletableResultCode export(Collection<SpanData> list) {
     StringBuilder stringBuilder = new StringBuilder();
     for (SpanData span : list) {
-
+      stringBuilder.setLength(0);
```

Comment:
Ouch, heh.

Are you interested in deleting this to switch to the SDK exporter?

Reply:
anything that gets us one span per line would be good for me. ;)